wremnants/datasets/datagroups.py

```diff
@@ -26,6 +27,10 @@ def __init__(self, infile, combine=False):
             self.rtfile = ROOT.TFile.Open(infile)
             self.results = None
 
+        if self.results:
+            self.dilepton = os.path.basename(self.results["meta_info"]["command"]).split()[0].startswith("mz")
```

Comment:
Is it ok for dilepton = True and like = True?

Reply:
yes, I made the dilepton flag as superset for wlike and dilepton since we often need the same setting for both